.github/workflows/build.yml

```diff
@@ -213,21 +205,23 @@ jobs:
         id: build-sdk
         run: task build
 
-      - name: Start the local node
-        id: start-local-node
-        if: ${{ !cancelled() && always() }}
+      - name: Prepare Hiero Solo
+        id: solo
+        uses: hiero-ledger/hiero-solo-action@71219540ac7f578e6ea4fc3c17575c0295e56163 # v0.7
```

Comment:
The lastest `hiero-solo-action` version is `v0.10.0` https://github.com/hiero-ledger/hiero-solo-action/releases